pkg/proto/modelzooserver.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package proto;
+
+message ModelRequest {
+    // name of the model definition or trained model.
+    // 1. For model definition, it could be a full Docker image URL like:
+    //    hub.docker.com/my_group/model_image_name, or path of the image to use the default registry:
+    //    my_group/model_image_name
+    // 2. For trained model, it's a unique string describes the model, like: my_resnet50_model_imagenet_acc_0.82
+    string name = 1;
+    // tag of the model definition or trained model, usually a vertion tag like: v0.1
+    string tag = 2;
+    // content_tar is the model definition's directory which contains a Dockerfile and
+    // other files that define some models.
+    // or, can be the content of a trained model.
+    bytes content_tar = 3;
```

Comment:
How comes the value `content_tar`? We need an explanation of the releasing process in the design doc.

Reply:
Please refer to  https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/design/model_zoo.md#release-a-trained-model